projects/angular/src/data/_tables.clarity.scss

```diff
@@ -43,8 +43,7 @@
       table-variables.$clr-table-font-color,
       variables.$clr-use-custom-properties
     );
-    margin: 0;
-    margin-top: variables.$clr_baselineRem_1;
+    margin: variables.$clr_baselineRem_1 0 0 0;
```

Comment:
Think about exposing this as a property in `_properties.tables.scss` although we don't need to change it in the shim.

Reply:
corrected in commit 69bb71b.